build.gradle

```diff
@@ -152,12 +153,79 @@ tasks.whenTaskAdded {task ->
     }
 }
 
+def splitTestConfig = [
+    dlicDlsflsTest: [
+        includeTestsMatching: [
+            "org.opensearch.security.dlic.dlsfls.*"
+        ]
+    ],
+    dlicRestApiTest: [
+        includeTestsMatching: [
+            "org.opensearch.security.dlic.rest.*"
+        ]
+    ],
+    crossClusterTest: [
+        includeTestsMatching: [
+            "org.opensearch.security.ccstest.*"
+        ]
+    ],
+    sslTest: [
+        includeTestsMatching: [
+            "org.opensearch.security.ssl.*"
+        ]
+    ],
+    securityIntegrationTest: [
+        includeTestsMatching: [
+            "org.opensearch.security.*Integ*"
+        ],
+        excludeTestsMatching: [
+            "org.opensearch.security.sanity.tests.*",
+            "org.opensearch.security.ssl.OpenSSL*",
+        ]
+    ],
+    indicesTest: [
+        includeTestsMatching: [
+            "org.opensearch.security.*indices*"
+        ],
+        excludeTestsMatching: [
+            "org.opensearch.security.sanity.tests.*",
+            "org.opensearch.security.ssl.OpenSSL*",
```

Comment:
I think they appeared here due to the `opensslTest` task

Reply:
Yes, I've checked that and we only needs `.sanity.tests.*` here. It's been already fixed.